package.json

```diff
@@ -13,14 +13,24 @@
   "author": "@pantheon-systems",
   "main": "index.js",
   "scripts": {
-    "test": "pnpm recursive test",
+    "build:all": "pnpm recursive run build",
```

Comment:
This might be better constructed as `pnpm build:pkgs && pnpm build:starters` since the starters have a dependency on the packages. This would also leave out docs from being built which I think is fine for right now.

Also, I know it was there before, but we can get rid of 'run' in all of the build commands.